assets/heatmap_header.txt

```diff
@@ -0,0 +1,12 @@
+#id: 'Sample Similarity'
+#title: 'Sample Similarity' 
+#section_name: 'edgeR: Sample Similarity' 
```

Comment:
I think this should be `Sample Similarity` and the `title` above isn't doing anything..?